covalent_ssh_plugin/ssh.py

```diff
@@ -156,7 +155,8 @@ def execute(
             info_dict = {"STATUS": Result.RUNNING}
             info_queue.put_nowait(info_dict)
 
-        ssh_success = self._client_connect()
+        ssh_success, conn = await self._client_connect()
```

Comment:
would it be possible here to check if a connection already exists?

Reply:
Yeah, we should be able to do that.